lesson07/assignment/index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+        <title>Activity 07 - Vue (blog page)</title>
+        <link rel="stylesheet" href="./styles/styles.css">
+        <link href="https://fonts.googleapis.com/css2?family=Noto+Sans&display=swap" rel="stylesheet">
+	<script src="https://kit.fontawesome.com/3f336ee7e9.js" crossorigin="anonymous"></script>
+        <script src="https://unpkg.com/vue@next"></script>
+    </head>
+    <body>
+        <!-- below is the app function for vue -->
+        <div id="app">
+            <!-- above is the app function for vue  -->
+        <a href="#main-content" class="hidden">Skip to main content</a>
+	<div class="content">
+	<!--insert contents here.-->
+	<header id="top">
+		<h1 class="pageheader">Family Blogs</h1>
+		<!-- <hr> -->
+	</header>
+	<ul class="ulNav">
+		<li class="linav">
```

Comment:
If these were wrapped in a `nav` tag, they wouldn't need these classes, could just use selectors of `nav ul` and `nav li` 😄